esptool.py

```diff
@@ -2213,6 +2214,15 @@ def version(args):
 # End of operations functions
 #
 
+def all_serial_ports():
```

Comment:
I think this function can be replaced with `list(sorted(list_ports.comports()))` ) or possibly just `sorted(list_ports.comports())`.

Please move the `import` to the import block at the top of the file.

Reply:
Will try and look at this today

Reply:
Put into a single line instead of a function